pkgs/servers/sql/mariadb/default.nix

```diff
@@ -208,6 +208,10 @@ in stdenv.mkDerivation (common // {
     "-DPLUGIN_AUTH_PAM=OFF"
     "-DWITHOUT_OQGRAPH=1"
     "-DWITHOUT_PLUGIN_S3=1"
+  ] ++ lib.optionals (stdenv.buildPlatform != stdenv.hostPlatform) [
+    # TODO: revisit this if nixpkgs supports any architecture whose stack grows upwards
```

Comment:
I don't think this should be a `TODO`, because it's not likely to be actionable to somebody reading this.